routes/student.routes.js

```diff
@@ -0,0 +1,113 @@
+const User = require("../models/User.model")
+
+const router = require("express").Router()
+
+const { sameUser, isLoggedIn, checkRoles } = require("../middleware")
+
+
+// DISPLAY ALL STUDENTS (only names)
+router.get('/', isLoggedIn, (req, res) => {
+
+    User
+    .find()
+    .select('username')
+    .then(theStudents => res.render('students/students', {theStudents}))
+    .catch(err => console.log(err))
+})
```

Comment:
Si es una lista de estudiantes agregaría esto:

```suggestion
// DISPLAY ALL STUDENTS (only names)
router.get('/', isLoggedIn, (req, res) => {

    User
    .find({role: "Student})
    .select('username')
    .then(theStudents => res.render('students/students', {theStudents}))
    .catch(err => console.log(err))
})
```